packages/core/src/index.ts

```diff
@@ -221,18 +232,19 @@ export const logHandledError = (
   error: Error,
   properties?: Properties,
 ): Promise<boolean> => {
-  // TODO REFACTOR WHEN iOS IMPLEMENT THE METHOD
-
-  // if (error instanceof Error) {
-  //   return NativeModules.EmbraceManager.logHandledError(
-  //     error.message,
-  //     error.stack,
-  //     properties,
-  //   );
-  // } else {
+  if (error instanceof Error) {
+    const {stack, message} = error;
+
+    return NativeModules.EmbraceManager.logHandledError(
+      message,
+      stack,
+      properties || {},
```

Comment:
@jpmunz fyi this is the change I mentioned in [slack](https://embrace-io.slack.com/archives/D040PBL681E/p1723833729929179)